tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -324,6 +330,50 @@ def fe_output_user_data_repack(input_model: InputModel, outputs: list, framework
     return _outputs
 
 
+def convert_params_lists_to_dicts(input_model, input_user_shapes, input_user_data_types, freeze_placeholder):
```

Comment:
can you please add description to this function? Really hard to understand what we expect and how this parses and packs output.